mods/LagSpikeOfDeath.json

```diff
@@ -0,0 +1,44 @@
+{
+    "format": 1,
+    "name": "LagSpikeOfDeath",
+    "authors": ["sp614x"],
+    "desc": "Minecraft optimization mod",
+    "versions": [
+        {
+            "name": "TODO_NAME",
+            "filename": "LagSpikeOfDeath_1.7.2.zip",
```

Comment:
I've just realized this field isn't in the spec, but I plan to add it, so keep it. :)